astroid/brain/brain_numpy_ndarray.py

```diff
@@ -120,7 +120,7 @@ def ptp(self, axis=None, out=None): return np.ndarray([0, 0])
         def put(self, indices, values, mode='raise'): return None
         def ravel(self, order='C'): return np.ndarray([0, 0])
         def repeat(self, repeats, axis=None): return np.ndarray([0, 0])
-        def reshape(self, shape, order='C'): return np.ndarray([0, 0])
+        def reshape(self, *shape, order='C'): return np.ndarray([0, 0])
```

Comment:
https://github.com/numpy/numpy/blob/9d7dca4596d7aeb8f31a261592a33d3b38b93b5f/numpy/__init__.pyi#L1767-L1774

The actual code is a bit different.. I'm not sure how to express this as it feels like we might run into issues with the other form as well..